substrate/frame/revive/fixtures/contracts/Deposit.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.20;
+
+contract Deposit {
```

Comment:
nit let's declare the precompile address here so we don't repeat it everywhere else down below

```
address immutable storagePrecompile = address(0x901);
```